samza-azure/src/main/java/org/apache/samza/system/azureblob/AzureBlobConfig.java

```diff
@@ -38,6 +38,8 @@ public class AzureBlobConfig extends MapConfig {
   // fully qualified class name of the AzureBlobWriter impl for the producer system
   public static final String SYSTEM_WRITER_FACTORY_CLASS_NAME = SYSTEM_AZUREBLOB_PREFIX + "writer.factory.class";
   public static final String SYSTEM_WRITER_FACTORY_CLASS_NAME_DEFAULT = "org.apache.samza.system.azureblob.avro.AzureBlobAvroWriterFactory";
+  public static final String SYSTEM_BLOB_CLIENT_BUILDER_FACTORY_CLASS_NAME = SYSTEM_AZUREBLOB_PREFIX + "azureclientbuilder.factory.class";
```

Comment:
should we add this to the docs? see https://github.com/apache/samza/blob/master/docs/learn/documentation/versioned/jobs/samza-configurations.md